src/shell/scripts/omp.bash

```diff
@@ -3,23 +3,30 @@ export POSH_SHELL_VERSION=$BASH_VERSION
 export POWERLINE_COMMAND="oh-my-posh"
 export POSH_PID=$$
 export CONDA_PROMPT_MODIFIER=false
+export OSTYPE=$OSTYPE
 
+# global variables
 _omp_start_time=""
 _omp_stack_count=0
 _omp_elapsed=-1
 _omp_no_exit_code="true"
 _omp_status_cache=0
 _omp_pipestatus_cache=0
+_omp_executable=::OMP::
+
+# switches to enable/disable features
+_omp_cursor_positioning=0
+_omp_ftcs_marks=0
 
 # start timer on command start
 PS0='${_omp_start_time:0:$((_omp_start_time="$(_omp_start_timer)",0))}$(_omp_ftcs_command_start)'
 # set secondary prompt
-PS2="$(::OMP:: print secondary --config="$POSH_THEME" --shell=bash --shell-version="$BASH_VERSION")"
+PS2="$(${_omp_executable} print secondary --shell=bash --shell-version="$BASH_VERSION")"
```

Comment:
This should be quoted to prevent word splitting after the variable expansion.

Reply:
@lewis-yeung can you provide a quick PR? Maybe the expansion isn't needed here.

Reply:
Sure. In a new PR, I'm going to include other changes that I suggested.